packages/components/src/ui/ariakit-tooltip/test/index.tsx

```diff
@@ -0,0 +1,330 @@
+		const user = userEvent.setup();
+
+		render( <ToolTip { ...props } /> );
+
+		await user.click( screen.getByRole( 'button', { name: /Button/i } ) );
+
+		expect(
+			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
+		).not.toBeInTheDocument();
+	} );
+
+	it( 'should respect custom delay prop when showing tooltip', async () => {
+		const user = userEvent.setup( { delay: TOOLTIP_DELAY } );
+		const CUSTOM_DELAY = TOOLTIP_DELAY + 25;
+
+		render( <ToolTip { ...props } delay={ CUSTOM_DELAY } /> );
+
+		const button = screen.getByRole( 'button', { name: /Button/i } );
+
+		await user.hover( button );
+
+		expect(
+			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
+		).not.toBeInTheDocument();
+
+		await waitFor( () =>
+			expect(
+				screen.getByRole( 'tooltip', { name: /tooltip text/i } )
+			).toBeVisible()
+		);
+
+		await user.unhover( button );
+
+		expect(
+			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
+		).not.toBeInTheDocument();
+
+		// Wait for the tooltip element to be positioned (aligned with the button)
+		// await waitFor( () =>
+		// 	expect(
+		// 		getWrappingPopoverElement( screen.getByText( 'tooltip text' ) )
+		// 	).toBePositionedPopover()
+		// );
+	} );
+
+	it( 'should show tooltip when an element is aria-disabled', async () => {
+		const user = userEvent.setup();
+
+		render(
+			<ToolTip { ...props }>
+				<Button aria-disabled>Button</Button>
+			</ToolTip>
+		);
+
+		const button = screen.getByRole( 'button', { name: /Button/i } );
+
+		expect( button ).toBeVisible();
+		//expect( button ).toBeDisabled();
+
+		await user.hover( button );
+
+		await waitFor( () =>
+			expect(
+				screen.getByRole( 'tooltip', { name: /tooltip text/i } )
+			).toBeVisible()
+		);
+
+		await user.unhover( button );
+
+		expect(
+			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
+		).not.toBeInTheDocument();
+
+		// Wait for the tooltip element to be positioned (aligned with the button)
+		// await waitFor( () =>
+		// 	expect(
+		// 		getWrappingPopoverElement( screen.getByText( 'tooltip text' ) )
+		// 	).toBePositionedPopover()
+		// );
+	} );
+
+	it( 'should not show tooltip if the mouse leaves the tooltip anchor before set delay', async () => {
+		const user = userEvent.setup();
+		const onMouseEnterMock = jest.fn();
+		const onMouseLeaveMock = jest.fn();
+		const MOUSE_LEAVE_DELAY = TOOLTIP_DELAY - 200;
+
+		render(
+			<>
+				<ToolTip { ...props }>
+					<Button
+						onMouseEnter={ onMouseEnterMock }
+						onMouseLeave={ onMouseLeaveMock }
+					>
+						Button 1
+					</Button>
+				</ToolTip>
+				<Button>Button 2</Button>
+			</>
+		);
+
+		await user.hover(
+			screen.getByRole( 'button', {
+				name: 'Button 1',
+			} )
+		);
+
+		// ToolTip hasn't appeared yet
+		expect(
+			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
+		).not.toBeInTheDocument();
+		//expect( onMouseEnterMock ).toHaveBeenCalledTimes( 1 );
```

Comment:
The assertions related to the mocks are commented out at the moment because they aren't passing. On my to-do list to look into.

Reply:
This has been fixed